packages/network/lib/cors.ts

```diff
@@ -22,7 +22,7 @@ export function parseDomain (domain: string, options = {}) {
   }))
 }
 
-export function parseUrlIntoDomainTldPort (str) {
+export function parseUrlIntoHostProtocolDomainTldPort (str) {
```

Comment:
Can we just call this `parse`?

Reply:
We can but it is used in `remote_states` and a few other places, which makes me wonder if it's cleared what is being `parse`d

Reply:
the `str`, duh 🤪

Reply:
Honestly I wonder if the type inference is enough?